### ISSUE TITLE
Language choice as the FIRST action when installing qTox

### ISSUE THREAD

**Issue by Saaaam** [qTox/qTox#3393]
Hello ;

In the following of https://github.com/tux3/qTox/issues/2758, where "By default we already try to use the system language, and fallback on English if it fails."
A solution could be the choice of the language as the first action from the user, at the installation. A window could be added at the beginning. The system language can be detected and the relative option selected. But if it isn't selected, the default option can be set on "English", still with the possibility to change for another option.

At the end of the installation, the software can use the language selected at first. It prevents the user to search how to change the language in the Preferences in a new software he/she doesn't know yet…

**Comment by zetok** [comment]
> "By default we already try to use the system language, and fallback on English if it fails."

Clearly doesn't work, and needs to be fixed.

**Comment by Saaaam** [comment]
Exactly, and this little language select box where the user has the last word about how the setup and the software are displayed could be a solution…

**Comment by zetok** [comment]
Not a solution, but a workaround.
